### ISSUE TITLE
string-template-indent changes content of string literal

### ISSUE THREAD

**Issue by mgroth0** [pinterest/ktlint#2530]
## Expected Behavior

Using `string-template-indent` should not change the content of a string literal

## Observed Behavior

Using `string-template-indent` is changing a the content of a string literal

## Steps to Reproduce

1. Enable only the following rules:

```
ktlint_standard = disabled # disable all other rules
ktlint_standard_indent = enabled
ktlint_standard_multiline-expression-wrapping = enabled
ktlint_standard_string-template-indent = enabled
```

2. Input the code below to the rule

```kotlin
fun bug() {
    val strings = listOf("a")
    println(
        """
${strings.joinToString { "a" }}
        """.trimIndent()
    )
}
```

3. Observe the output is the following:

```
fun bug() {
    val strings = listOf("a")
    println(
        """
        ${strings.joinToString { "a" }}strings.joinToString { "a" }}
        """.trimIndent()
    )
}
```

## Your Environment
* Version of ktlint used: 1.1.1
* Using the API directly

**Comment by paul-dingemans** [comment]
Tnx for reporting. This is clearly a bug.

The problem only seem to occurr when the line starts with a string template. Given code below:
```
fun bug() {
    val strings = listOf("a")
    println(
        """
${strings.joinToString { "a" }}
 ${strings.joinToString { "a" }}
$strings
 $strings
strings
        """.trimIndent(),
    )
}
```
it is formatted as:
```
fun bug() {
    val strings = listOf("a")
    println(
        """
        ${strings.joinToString { "a" }}strings.joinToString {
            "a"
        }}
         ${strings.joinToString { "a" }}
        $stringsstrings
         $strings
        strings
        """.trimIndent(),
    )
}
```

**Comment by paul-dingemans** [comment]
Actually there are two different problems:
* If the line starts with a string template then the content of that string template is duplicated on the same line
* The content of the raw string literal should not be indented when it starts at the first position of the line

**Comment by paul-dingemans** [comment]
Resolved by #2553
